mdx-layout/src/StylesheetSwitch.tsx

```diff
@@ -29,9 +29,11 @@ const labelStyle: CSSProperties = {
 export const StylesheetSwitch = ({
   defaultColorScheme = 'light',
   stylesheets,
+  onSwitch = () => {},
 }: {
   defaultColorScheme?: ColorScheme;
   stylesheets: Record<ColorScheme, string>;
+  onSwitch?: (scheme: 'dark' | 'light') => void;
```

Comment:
you can type with ColorScheme here I think

Reply:
fixed